deployment/.helmignore

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018-2021 Crown Copyright
```

Comment:
again, why 2 different helmignores? doesnt one in the root pal-examples work?

Reply:
Removed as there is one in the root directory now.